Documentation/kubernetes-networking.md

```diff
@@ -0,0 +1,101 @@
+# Kubernetes Networking
+
+## Network Model
+
+The Kubernetes network model outlines four methods of component communication:
+
+* Container-to-Container Communication
```

Comment:
I'm not convinced we need the Container-to-Container documentation at all. Especially below, it just feels unnecessary.